examples/local-example.py

```diff
@@ -93,7 +92,7 @@ def main():
     # Create actor
     actor = ExampleActor(client, modifier)
     # Start work!
-    actor.run()
+    actor.run(max_tasks=2, stop_function=actor.time_elapsed, elapsed=11)
```

Comment:
I find the arguments `stop_function` and `elapsed` confusing. It seems that this has the same functionality as `max_time` but for some reason `max_time` is broken (as is commented in the `time_elapsed` function). Why not fix `max_time` by moving `time_elapsed` function to actors.py and use it directly in `actors.run()` (without needing it to pass as an argument), so that the user only needs to use the `max_time` argument in local_example.py. It would be best to keep `local_example.py` as simple as possible.

Reply:
The `stop_function` can take an arbitrary boolean function, meaning that the user can use any condition to stop at any time. The elapsed is passed into the `actor.time_elapsed` through the `**kwargs`.

This is an example of using `stop_function`, where the choice of using the time instead of something else may be unfortunate as it is very similar to the `max_time`.